aws_deploy/cache.tf

```diff
@@ -0,0 +1,53 @@
+resource "aws_cloudfront_distribution" "main" {
+  enabled         = true
+  is_ipv6_enabled = true
+  price_class     = "PriceClass_100"
+  aliases         = terraform.workspace == "production" ? [local.domain_root, local.domain_frontend] : [local.domain_frontend]
```

Comment:
This is not working. We need a slightly different approach: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/CNAMEs.html#alternate-domain-names-requirements